engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -956,10 +956,18 @@ static void WebGPUDeleteContext(HContext _context)
 static void WebGPURunApplicationLoop(void* user_data, WindowStepMethod step_method, WindowIsRunning is_running)
 {
     TRACE_CALL;
+#ifdef __EMSCRIPTEN__
+    while (0 != is_running(user_data))
+    {
+        // N.B. Beyond the first test, the above statement is essentially formal since set_main_loop will throw an exception.
+        emscripten_set_main_loop_arg(step_method, user_data, 0, 1);
+    }
+#else
```

Comment:
Please put unrelated changes in different pull requests, thanks! :)

Reply:
Sorry, didn't mean to push.